Tests/UnitTests/Screens/Home/ServicesTileViewModelTests.swift

```diff
@@ -0,0 +1,56 @@
+import GDSAnalytics
+import GDSCommon
+@testable import OneLogin
+import XCTest
+
+@MainActor
+final class ServicesTileViewModelTests: XCTestCase {
+    var mockAnalyticsService: MockAnalyticsService!
+    var sut: ServicesTileViewModel!
+    
+    var didCallButtonAction = false
+    
+    override func setUp() {
+        super.setUp()
+        
+        mockAnalyticsService = MockAnalyticsService()
+        sut = ServicesTileViewModel(analyticsService: mockAnalyticsService) {
+            self.didCallButtonAction = true
+        }
+    }
+    
+    override func tearDown() {
+        mockAnalyticsService = nil
+        sut = nil
+        
+        didCallButtonAction = false
+        
+        super.tearDown()
+    }
+}
+
+extension ServicesTileViewModelTests {
```

Comment:
Would be great if we can add a quick test for the callback too: 
<img width="663" alt="image" src="https://github.com/user-attachments/assets/c8bf945b-d380-4228-be6b-04b1af387932">